examples/next/app-router/.env.local.sample

```diff
@@ -1,5 +1,8 @@
 # Your WordPress site URL
 NEXT_PUBLIC_WORDPRESS_URL=https://faustexample.wpengine.com
 
+# Your Faust site URL
+NEXT_PUBLIC_URL=http://localhost:3000
```

Comment:
Using `fetch` in node does not allow relative URLs. Since we are using `fetch` now within RSCs, we need to know the site URL.